### PR TITLE
FAQs related to data - Update faq.json with EN articles

### PR DESCRIPTION
**Pull request by dsarkar**

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -1542,6 +1542,38 @@
...
+                            "The Exposure Notification Framework (ENF) keys are managed by the operating system and can be deleted in the system settings of Android or iOS (See  <a href='#delete_all_data_cwa' target='_self' rel='noopener noreferrer'>See How can I delete all data stored in the CWA?</a>)."
+                        ]
+                    },
+                    {
+                        "title": "How can I delete all data stored in the Corona-Warn-App?",
+                        "anchor": "delete_all_data_cwa",
+                        "active": true,
+                        "textblock": [
+                            "By de-installing the Corona-Warn-App (CWA), all data of the CWA is deleted. However, a user should note that, for example, the data of the Exposure Notification Framework (ENF) is managed by the operating system of his smartphone. Therefore, this ENF data on Android is only deleted by a CWA deinstallation if no other application (app) has been installed on the user's smartphone that uses the ENF. On Apple iOS, the ENF data must be deleted manually because it will not be deleted by the CWA deinstallation.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1562]: "de-installing" => "uninstalling"
"a CWA deinstallation" => "uninstalling CWA"
"the CWA deinstallation" => "uninstalling CWA"

Android uses the term "uninstall"
https://dictionary.cambridge.org/dictionary/english/uninstall

**Review by MikeMcC399 — COMMENTED** [review]
Also for the English version, I could not find the verb "to deinstall" in the dictionary. Android does not use this term either. It calls it "uninstall".

**Comment by dsarkar** [comment]
Good morning @MikeMcC399 Thank you very much! I will correct the article.

File: src/data/faq.json (hunk 1 above)
Comment (dsarkar) [reply src/data/faq.json:1562]: Fix will follow. Thanks.

**Pull request merged**